src/slider/style.scss

```diff
@@ -29,6 +29,18 @@ tp-slider-slide {
 	}
 }
 
+.tp-slider--grabbing {
+	cursor: grabbing;
+}
+  
+.tp-slider--grab {
+	cursor: grab;
+}
+  
+.tp-slider--dragging {
```

Comment:
@AvinashVinod03 lets use javascript to do the style updates, lets remove these from here.